test/unit/rules/block-indentation-test.js

```diff
@@ -687,4 +687,18 @@ generateRuleTests({
       ],
     },
   ],
+  errors: [
+    {
+      template: `
+      {{#ach this.foo as |bar|}}
+        <li>{{bar}}</li>
+      {{/each}}`,
+      results: [
+        {
+          fatal: true,
+          message: "ach doesn't match each - 3:9",
```

Comment:
This linting error isn't related to the rule. Do you mind moving this test scenario to [`test/acceptance/public-api-test.js`](https://github.com/ember-template-lint/ember-template-lint/blob/v2.4.0/test/acceptance/public-api-test.js#L324)? This is where the `verify` function is tested.

Changes looks good to me otherwise 👍

Reply:
Ya, agree.

@lifeart - Ping me when you've had a chance to update that, I'll try to land + release...